src/gui/src/settings/options-window.h

```diff
@@ -86,10 +85,10 @@ class OptionsWindow : public QDialog
 		void addFilename(const QString &, const QString &, const QString &);
 		void on_buttonMetadataPropsysAdd_clicked();
 		void on_buttonMetadataExiftoolAdd_clicked();
-		void checkSpinners(int);
+		void checkSpinners(int newVal);
 		//void checkAllSpinners(QList<std::pair<QCheckBox*, QSpinBox*>> *);
 		void checkAllSpinners();
-		void checkAllSpinnersWithPlacement(int);
+		void checkAllSpinnersWithPlacement(int srcPlacement);
```

Comment:
I actually left out the variable names because I thought that's what you preferred. I noticed many places where you did not include them, like like 85 just above this. What is the difference between these functions/parameters?

Reply:
This file is quite old so there's some old auto-generated code that don't have them, but it's better for IDEs and other things.